konfera/tests/test_models.py

```diff
@@ -134,22 +134,22 @@ def test_string_representation(self):
         )
 
     def test_automatic_order_generator(self):
+        time = timezone.now()
         location = models.Location(title='test_title', street='test_street', city='test_city', postcode=000000,
                                    state='test_state', capacity=20)
         location.save()
-        event = models.Event(title='test_event', description='test', event_type='meetup', status='published',
-                             location=location, date_from=timezone.now(), date_to=timezone.now())
+        event = models.Event(title='test_event', description='test', event_type='meetup',
+                             status=models.event.PUBLISHED, location=location, date_from=time, date_to=time)
```

Comment:
Probably better to import at the top of the file and use only constant here, also I noticed event_type above can used const as well...